docs/access.md

```diff
@@ -1,20 +1,118 @@
 # Access management
 
-## LaminDB
+This document outlines the access control system, detailing its architecture, key features, roles, and implementation guidelines.
 
-LaminDB doesn't come with access management but relies on the underlying data infrastructure for this.
+## System architecture
 
-## LaminHub
+Our access management is built on a four-tier hierarchy, providing granular control over resources:
 
-```{include} includes/access-laminhub.md
+1.  **Organizations:** The top-level entity. Contains all associated teams, instances, and spaces. Acts as the primary boundary for users and resources.
+2.  **Teams:** Groups of users within an organization. Roles and permissions can be assigned collectively to teams, simplifying management.
+3.  **Instances:** Collaborative workspaces (e.g., projects, environments) within an organization. Each instance has its own permission structure and collaborators.
+4.  **Spaces:** Fine-grained access control areas *within* instances. Allow specific permissions to be set on subsets of data or resources within an instance (e.g., datasets, models, notebooks).
 
-```
 
-## Cloud providers
+## Key features
 
-Currently, only on the enterprise plan, we're happy to
+### Spaces
 
-1. host databases & storage in Lamin's AWS/GCP infrastructure
-2. help you connect LaminHub to databases & storage in your AWS/GCP infrastructure
+Spaces are areas within instances that allow specific permissions to be set on subsets of data for organization resources by department, project, or sensitivity level:
+* Each space has its own set of collaborators with their roles and permissions, independent of instance-level roles. 
+* Users must be both instance collaborators AND space collaborators to access resources in a space within an instance
+* Spaces must be attached to an instance before records from that instance can be assigned to the space (you need both instance and space admin permissions to create this attachment)
+* Spaces are applied at the database record level, and a record can only belong to one space
 
-We'll role out a fully hosted Pro plan over time.
+<div align="center">
+  <img src="./access_spaces.png" style="width: 90%;"/>
+</div>
+
+### All space
+
+*   **Default "All" space:** Every instance includes a default `"All"` space upon creation. This space typically holds common, non-sensitive resources.
```

Comment:
This space holds common resources that are meant to be accessible to all instance collaborators.